src/shared/components/community/sidebar.tsx

```diff
@@ -767,3 +768,28 @@ export class Sidebar extends Component<SidebarProps, SidebarState> {
     );
   }
 }
+
+export function community_visibility_label(
+  visibility: CommunityVisibility,
+): NoOptionI18nKeys {
+  var visibility_label = "community_visibility_";
```

Comment:
Don't use var. Just use early return and remove all the breaks.

Reply:
That would be much more verbose as it needs to convert to `NoOptionI18nKeys` every time.

Reply:
Then use `let`.

`var` is deprecated in typescript, only use let or const. This could also be done with a `const Map` also. Look at `sort-select.tsx` if you want to see a good example of how that's done.